### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by misheck12**
In this project i setup the backend for the project

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    name: Rubocop
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 3.0.x
+      - name: Setup Rubocop
+        run: |
+          gem install --no-document rubocop -v '>= 1.0, < 2.0' # https://docs.rubocop.org/en/stable/installation/
```
Comment (keza681) [inline .github/workflows/linters.yml:19]: Hello @misheck12, you did a wonderful work actually, I really appreciate it. Unfortunately I can not proceed by approving the work as the linters are falling. Kindly check them out and push again. I hope you will make it 👌
Happy coding 😺

**Review by keza681 — CHANGES_REQUESTED** [review]
Hi @misheck12 

Your project needs a change highlighted below!  
## Highlights
- Linters errors ❌ 
- Git flow was used correctly 🟢 
- Very good configuration 🟢 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @keza681

File: .github/workflows/linters.yml (hunk 1 above)
Comment (misheck12) [reply .github/workflows/linters.yml:19]: sure

Comment (misheck12) [reply .github/workflows/linters.yml:19]: hello @keza681 i have fixed the errors please review this

**Review by keza681 — APPROVED** [review]
# STATUS: APPROVED 🟢 🎉🎉🎉

### Hi @misheck12 

Great job implementing your part of the project 👍🏽  .
Your project is complete!  There is nothing else to say other than... it's time to merge it  :shipit: 
**Congratulations! 🎉** 
Gu2dpSp0qee4C710M5kVsWN425J3-vv2w52by6
## To Highlight

- linters are passing well
- Good commit history
- Good PR description

----------------------------------------------------------------------------------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @keza681  in your question so I can receive the notification.

File: .github/workflows/linters.yml (hunk 1 above)
Comment (keza681) [reply .github/workflows/linters.yml:19]: Hi @misheck12, well done. Can you merge it now?

**Pull request merged**